packages/reports/addon/helpers/should-show-id.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Helper to determine if we sould see the id field, next to name/description in a list.
+ * Now supports arguments list that are shaped for powerSelect grouped content.
+ */
+import { helper } from '@ember/component/helper';
+
+type Arg = {
+  name?: string;
+  description?: string;
+  groupName?: string;
+  options?: Array<Arg>;
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function shouldShowId(_unused: any[], { argument, params }: { argument: Arg; params?: Arg[] }): any {
+  if (!Array.isArray(params)) {
+    return true;
+  }
+  const paramArray = params as Array<Arg>;
```

Comment:
After tsconfig update, we can remove the `as __` casts and this intermediate variable